api/src/main/java/org/pmiops/workbench/auth/ServiceAccounts.java

```diff
@@ -7,11 +7,13 @@
 import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
 import java.util.List;
+import org.springframework.stereotype.Component;
 
 /**
  * Handles functionality related to loading service account credentials and generating derived /
  * impersonated credentials.
  */
+@Component
```

Comment:
What's the significance of adding this? Isn't this still just a collection of static utilities both before/after this change?